app/src/main/java/org/mozilla/tv/firefox/ScreenController.kt

```diff
@@ -94,6 +94,7 @@ class ScreenController(private val sessionRepo: SessionRepo) {
         val transition = when (settingsScreen) {
             SettingsScreen.DATA_COLLECTION -> Transition.ADD_SETTINGS_DATA
             SettingsScreen.CLEAR_COOKIES -> Transition.ADD_SETTINGS_COOKIES
+            SettingsScreen.FXA_PROFILE -> Transition.ADD_FXA_PROFILE
```

Comment:
I agree that this could be cleaner, but after spending some time fiddling with these states, I think it would be too much work to approach this in another way.  Unless you have strong opposition, I'd prefer to keep it as is.

Reply:
I agree that this could be cleaner, but after spending some time fiddling with these states, I think it would be too much work to approach this in another way. Unless you have strong opposition, I'd prefer to keep it as is.